test/verify/check-system-journal

```diff
@@ -457,6 +469,33 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
                         "Load earlier entries"
                         ])
 
+        # Debian supports grep only from 19.10, see https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1751006
```

Comment:
19.10 is an Ubuntu release version number, not a [Debian](https://www.debian.org/releases/) one. That would be "Debian 10" (== debian-stable)